### DISCUSSION TITLE
CVE-2024-37897

### DISCUSSION THREAD

**Discussion by thielj** [drakkan/sftpgo#1664] [General]
Are there any details available?

**Comment by thielj** [comment]
Looks like it has been published now

https://github.com/drakkan/sftpgo/security/advisories/GHSA-hw5f-6wvv-xcrh

**Comment by drakkan** [comment]
Hello,

Please understand that publishing a CVE requires some technical time. Details were published the day after v2.6.1 was released.

In any case, people who care about SFTPGo and/or the security of SFTPGo and therefore of their production installations, can purchase a support contract, in this way they will ensure that the project stays alive and they will get security hardening, early security patches and many other features. More [details](https://sftpgo.com/#pricing). Thanks

**Reply by thielj** [reply]
Apologies for reading the release notes. I'll try to avoid installing new bits until I can afford a support contract for my hobby server.
